light-zk.js/src/account.ts

```diff
@@ -397,8 +427,8 @@ export class Account {
     return privateKey;
   }
 
-  static generateAesSecret(seed: String, _domain?: string): Uint8Array {
-    const privkeySeed = seed + "aes";
+  static generateSecret(seed: String, domain: String): Uint8Array {
```

Comment:
nice!

Reply:
yeah, sprinkles of optimisation here and there :)